ci/kind/kind-upgrade.sh

```diff
@@ -0,0 +1,47 @@
+#!/usr/bin/env bash
+
+# Copyright 2020 Antrea Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -eo pipefail
+KIND_VERSION="v0.20.0"
+
+
+function upgrade_kind {
+    runtimeOS="$(uname -m)"
+    case "$runtimeOS" in
+     "x86_64")
+      curl -Lo ./kind https://kind.sigs.k8s.io/dl/v0.20.0/kind-linux-amd64
+      ;;
+      "aarch64")
+      curl -Lo ./kind https://kind.sigs.k8s.io/dl/v0.20.0/kind-linux-arm64
+      ;;
+    esac
+    chmod +x ./kind
+    sudo mv ./kind /usr/local/bin/kind
+    kind_version=$(kind version | awk  '{print $2}')
+    echo "$kind_version"
+}
+
+kind_version=$(kind version | awk  '{print $2}')
+kind_version=${kind_version:1} # strip leading 'v'
+function version_lt() { test "$(printf '%s\n' "$@" | sort -rV | head -n 1)" != "$1"; }
+if version_lt "$kind_version" "0.20.0" && [[ "$KUBE_PROXY_MODE" == "none" ]]; then
+    # This patch is required when using Antrea without kube-proxy:
+    # https://github.com/kubernetes-sigs/kind/pull/2375
+    echoerr "You have kind version v$kind_version installed"
```

Comment:
these two can be combined echoerr "You have kind version v$kind_version installed. You need to upgrade to kind >= v0.20.0 when disabling kube-proxy."